recording-daemon/notify.c

```diff
@@ -44,6 +45,9 @@ static void do_notify(void *p, void *u) {
 	if (!c)
 		goto fail;
 
+	curl_mime *mime;
+	mime = curl_mime_init(c);
```

Comment:
It would be, but i init mime here to make free with `curl_mime_free` after executing `curl_easy_perform` below. Actualy, I forget `curl_mime_init` on condition.

Reply:
Sure, but if condition `if (notify_record)` is met, then `mime = curl_mime_init(c)` is done twice, with the second one overwriting the return value of the first (without a matching `curl_mime_free` in between), resulting in a leak.

I would suggest to initialise it to NULL and then do `curl_mime_free` below based on `if (mime)`